test/cmdlineTests/standard_model_checker_solvers_cvc4/output.json

```diff
@@ -0,0 +1,5 @@
+{"errors":[{"component":"general","errorCode":"7649","formattedMessage":"Warning: CHC analysis was not possible since no Horn solver was enabled.
+
+","message":"CHC analysis was not possible since no Horn solver was enabled.","severity":"warning","type":"Warning"},{"component":"general","errorCode":"7710","formattedMessage":"Warning: BMC analysis was not possible since no SMT solver was found and enabled.
+
+","message":"BMC analysis was not possible since no SMT solver was found and enabled.","severity":"warning","type":"Warning"}],"sources":{"A":{"id":0}}}
```

Comment:
Exactly, it won't. At first I wanted to indeed make it succeed, and in most cases it actually works, that being my local setup + CI's Linuxes. But again, MacOS creates problem there. I didn't find an easy way of installing CVC4 there. I did manage to install it when ssh`ed into CI, but it takes very long and the job fails. I didn't feel like spending much time fixing cvc4 in CI and instead just removed it here.

Reply:
I mean: will I still get this warning with cvc4 installed so that the test passes? Or will this test fail when it's present?

From what you said, I understand it would be the former, which is fine. I only see a problem here if it requires you to uninstall `cvc4` to have this pass.

Reply:
The test will fail if you have cvc4, that's why I uninstalled from docker.

Reply:
Then I think this is a problem for local development. We do say in installation instructions that `cvc4` is optional. Now it becomes undesired. We could mention that for command-line tests to pass the compiler must be built with `-DUSE_CVC4=OFF` or `cvc4` must not be present - but maybe it would be better to just drop this particular test?

Reply:
But isn't it the same as the latest z3 requirement to run the tests? But yea, agree it's a weird requirement to *not* have something. Will remove the test.

Reply:
Well, it is the same type of restriction but having latest z3 does not interfere with development while this one could be pretty annoying when working on stuff that requires cvc4 :) Also, there's no `--no-smt` flag for command-line tests that would let you just skip test like this one that have special requirements.